packages/i18n/src/translateToExclusiveList.js

```diff
@@ -45,7 +45,8 @@ function translateToExclusiveList(options) {
         /* translators: %1$s: is a comma separated list. %2$s: last entry in list  */
         __('%1$s, or %2$s', 'web-stories'),
         options.slice(0, options.length - 1).join(
-          /* translators: delimiter used in a list */
+          /* eslint-disable-next-line @wordpress/i18n-no-flanking-whitespace -- Expected behaviour.
+           * translators: delimiter used in a list */
           __(', ', 'web-stories')
```

Comment:
I don't think this is gonna work. `translators` comments need to be on their own, otherwise they won't be picked up.

This should be safer:

```suggestion
          /* translators: delimiter used in a list */
          __(', ', 'web-stories') // eslint-disable-line @wordpress/i18n-no-flanking-whitespace -- Expected behaviour.
```